test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj

```diff
@@ -301,3 +315,103 @@
           "Unexpected port for second connector")
       (is (= 42 (.getSoLingerTime (second connectors)))
           "Unexpected so linger time for second connector"))))
+
+(deftest create-server-idle-timeout-test
+  (testing "idle-timeout configured properly for http connector"
+    (let [server (jetty/create-server
+                   (get-webserver-context-for-server)
+                   (get-http-config-for-server 0
+                                               0
+                                               0
+                                               3000))
+          connectors (.getConnectors server)]
+      (is (= 1 (count connectors))
+          "Unexpected number of connectors for server")
+      (is (= 3000 (.getIdleTimeout (first connectors)))
+          "Unexpected idle time for connector")))
+  (testing "idle-timeout configured properly for multiple connectors"
+    (let [server (jetty/create-server
+                   (get-webserver-context-for-server)
+                   {:http {:port 25
+                           :host "0.0.0.0"
+                           :request-header-max-size 100
+                           :so-linger-milliseconds 41
+                           :idle-timeout-milliseconds 9001}
+                    :https {:port 92
+                            :host "0.0.0.0"
+                            :protocols nil
+                            :cipher-suites nil
+                            :keystore-config
+                            {:truststore (-> (KeyStore/getDefaultType)
+                                             (KeyStore/getInstance))
+                             :key-password "hello"
+                             :keystore (-> (KeyStore/getDefaultType)
+                                           (KeyStore/getInstance))}
+                            :request-header-max-size 100
+                            :so-linger-milliseconds 42
+                            :idle-timeout-milliseconds 9002
+                            :client-auth :want}
+                    :jmx-enable false
+                    :max-threads 100
+                    :queue-max-size 101})
+          connectors (.getConnectors server)]
+      (is (= 2 (count connectors))
+          "Unexpected number of connectors for server")
+      (is (= 25 (.getPort (first connectors)))
```

Comment:
Oh, I see - it's because we explicitly add the HTTP connector first, and then the HTTPS connector.  And `getConnectors` returns an array.  That's good enough for me.